src/toil/test/lib/test_conversions.py

```diff
@@ -87,68 +87,56 @@ def test_convert(self):
-            '0.1 T': 109951162777,
-            '0.5 K': 512,
-            '0.5 M': 524288,
-            '0.5 G': 536870912,
-            '0.5 T': 549755813888,
-            '0.9 K': 921,
-            '0.9 M': 943718,
-            '0.9 G': 966367641,
-            '0.9 T': 989560464998,
-            '1 K': 1024,
-            '1 M': 1048576,
-            '1 G': 1073741824,
-            '1 T': 1099511627776,
-            '7 K': 7168,
-            '7 M': 7340032,
-            '7 G': 7516192768,
-            '7 T': 7696581394432,
-            '7.42423 K': 7602,
-            '7.42423 M': 7784869,
-            '7.42423 G': 7971706261,
-            '7.42423 T': 8163027212283,
-            '10 K': 10240,
-            '10 M': 10485760,
-            '10 G': 10737418240,
-            '10 T': 10995116277760,
-            '100 K': 102400,
-            '100 M': 104857600,
-            '100 G': 107374182400,
-            '100 T': 109951162777600,
-            '1000 K': 1024000,
-            '1000 M': 1048576000,
-            '1000 G': 1073741824000,
-            '1000 T': 1099511627776000,
-            '11234234 K': 11503855616,
-            '11234234 M': 11779948150784,
-            '11234234 G': 12062666906402816,
-            '11234234 T': 12352170912156483584
+            '0 Ki': 0,
+            '0 Mi': 0,
+            '0 Gi': 0,
+            '0 Ti': 0,
+            '0.1 Ki': 102,
+            '0.1 Mi': 104857,
+            '0.1 Gi': 107374182,
+            '0.1 Ti': 109951162777,
+            '0.5 Ki': 512,
+            '0.5 Mi': 524288,
+            '0.5 Gi': 536870912,
+            '0.5 Ti': 549755813888,
+            '0.9 Ki': 921,
+            '0.9 Mi': 943718,
+            '0.9 Gi': 966367641,
+            '0.9 Ti': 989560464998,
+            '1 Ki': 1024,
+            '1 Mi': 1048576,
+            '1 Gi': 1073741824,
+            '1 Ti': 1099511627776,
+            '7 Ki': 7168,
+            '7 Mi': 7340032,
+            '7 Gi': 7516192768,
+            '7 Ti': 7696581394432,
+            '7.42423 Ki': 7602,
+            '7.42423 Mi': 7784869,
+            '7.42423 Gi': 7971706261,
+            '7.42423 Ti': 8163027212283,
+            '10 Ki': 10240,
+            '10 Mi': 10485760,
+            '10 Gi': 10737418240,
+            '10 Ti': 10995116277760,
+            '100 Ki': 102400,
+            '100 Mi': 104857600,
+            '100 Gi': 107374182400,
+            '100 Ti': 109951162777600,
+            '1000 Ki': 1024000,
+            '1000 Mi': 1048576000,
+            '1000 Gi': 1073741824000,
+            '1000 Ti': 1099511627776000,
+            '11234234 Ki': 11503855616,
+            '11234234 Mi': 11779948150784,
+            '11234234 Gi': 12062666906402816,
+            '11234234 Ti': 12352170912156483584
         }
 
         results = {}
         for i in (0, 0.1, 0.5, 0.9, 1, 7, 7.42423, 10, 100, 1000, 11234234):
-            for src_unit in ['K', 'M', 'G', 'T']:
+            for src_unit in ['Ki', 'Mi', 'Gi', 'Ti']:
```

Comment:
Can we add `b` and the decimal counterparts?  So: `['b', 'Ki', 'Mi', 'Gi', 'Ti', 'K', 'M', 'G', 'T']`